### ISSUE TITLE
Tags and Labels Demo

### ISSUE THREAD

**Issue by mcarrano** [patternfly/patternfly-next#759]
Provide examples and guidelines for using tags and labels including:
- Common CRUD use cases
- Tags and labels in context

**Comment by mcarrano** [comment]
@rachael-phillips @LHinson I'm not seeing this anywhere on the roadmap. It should be.

**Comment by rachael-phillips** [comment]
Hi @mcarrano how high priority do you think this is and where would you like to see this on the roadmap?

**Comment by mcarrano** [comment]
@rachael-phillips I think this is something that has been a long standing need, and I know that many products make use of tagging. That said, we are not working from a specific product ask. I would call this medium priority. @LHinson what do you think? This would be a good item to place on the Wednesday mtg agenda.

**Comment by rachael-phillips** [comment]
@mcarrano yes, I will definitely pull this in to our Wednesday meeting agenda. Thank you for your feedback on this!

**Comment by rblackbu** [comment]
**Documentation: PatternFly 4 Component - Tagging** 
https://docs.google.com/document/d/1GGQx0BsuuA22waEF1OyN7nodpzO2DTvtPhAPX6dHCZ0/edit#heading=h.ogckf4qtmbqx

**Wireframe: Visuals and research**
https://redhat.invisionapp.com/d/main#/console/16693511/357761389/preview

**Comment by mcarrano** [comment]
Hey @rblackbu I'm noting that the link here is now stale since we've moved from InVision to Marvel.

**Comment by mcarrano** [comment]
Capturing some notes and mockups from discussion with the Ansible team.

> Our current use case calls for a [ X More ] label if there are more than 5 tags (apart from search filters). We also display a [ Show Less ] label as a toggle to the X More. We display closable tags in places like search filters, but we also display read only tags in a read-only details view. Here are some examples:

Closable Tags: https://tower-mockups.testing.ansible.com/patternfly/orgs/orgs/
Read-Only Tags with X More: https://tower-mockups.testing.ansible.com/patternfly/orgs/orgs-detail/
Show Less Toggle: https://tower-mockups.testing.ansible.com/patternfly/orgs/orgs-detail-more/

**Comment by rblackbu** [comment]
@mcarrano FYI - I updated the link to point to the prototype in Marvel.

**Comment by matthewcarleton** [comment]
There is a similar use case in OpenShift where currently they have created their own "tags" 
<img width="596" alt="Screen Shot 2019-08-07 at 12 11 05 PM" src="https://user-images.githubusercontent.com/287880/62638144-34087f00-b913-11e9-9a5b-041af8e8266f.png">
It could be nice to align with this proposal, it offers some nice interactions in terms of CRUD.
